dms_attachment_link/static/src/scss/dms_attachment_link.scss

```diff
@@ -0,0 +1,4 @@
+.o_AttachmentBox_content {
+    text-align: center;
+    display: block;
```

Comment:
I'd rather use bootstrap classes over the main template

![image](https://user-images.githubusercontent.com/5040182/222100205-d80b3ea2-ef4b-49c5-92b5-bf2e20b32b29.png)